.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
https://github.com/mafintosh/dns-packet/commit/e81b4020c0985014819ebdcc02334e7948d1344c has restriced CI runs to the master branch only, e.g. it would not run on PRs any more. I'm not sure if that was the intention behind that change, so I reinstated running on PR.